components/elm/src/main/subgridAveMod.F90

```diff
@@ -1009,6 +1010,11 @@ subroutine c2g_1d(bounds, carr, garr, c2l_scale_type, l2g_scale_type)
     garr(bounds%begg : bounds%endg) = spval
     sumwt(bounds%begg : bounds%endg) = 0._r8
     do c = bounds%begc,bounds%endc
+       !NaNs during initialization cause problems with debugger.
+       if (isnan(carr(c))) then
+          call endrun(msg='carr(c) is NaN '//&
+                errMsg(__FILE__, __LINE__))
+       endif
```

Comment:
`SMS.r05_r05.IELM.chrysalis_intel.elm-topounit_im2` is failing because of this NaN check. We should not have NaNs floating around in a simulation. I will run some more tests to know more.